java/idl2jni/codegen/im_java.cpp

```diff
@@ -217,6 +217,8 @@ std::string param_type(AST_Type *t, AST_Argument::Direction dir)
     case AST_PredefinedType::PT_boolean:
     case AST_PredefinedType::PT_char:
     case AST_PredefinedType::PT_wchar:
+    case AST_PredefinedType::PT_int8:
+    case AST_PredefinedType::PT_uint8:
```

Comment:
I think I forgot about this earlier, but you will need macro checks around `PT_int8` and `PT_uint8`. Like here https://github.com/objectcomputing/OpenDDS/blob/ef88d4e8561b8e3bbad42b953118949a2322f6b2/dds/idl/langmap_generator.cpp#L138-L141

This is where you include `dds/DCPS/Definitions.h`.